packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -23,49 +23,88 @@ import './page.css';
+
+### Vertical navigation
+
+To add vertical side navigation to a `<Page>`, pass a `<PageSidebar>` component into the `sidebar` property. To render navigation in the `<PageSidebar>`, use the `nav` property. 
+
+The `isNavOpen` property helps facilitate the opening and closing of the sidebar and should be 'true' when the navigation sidebar is visible. 
 
 ```ts file="./PageVerticalNav.tsx"
 ```
 
-### Horizontal nav
+### Legacy vertical navigation
+
+This example shows the legacy implementation of a page's vertical navigation. Our updated recommendation advises you to use a masthead and toolbar to make headers, rather than `<PageHeader>` and `<PageHeaderTools>` as shown in the following example.
+
+```ts file="./PageVerticalNavUsingPageHeaderComponent.tsx"
+```
+
+### Horizontal navigation
+
+To add horizontal top navigation to a `<Page>`, each navigation item should be set as a `<ToolbarItem>` in the `<Toolbar>` that is passed to the `<Masthead>`.
 
 ```ts file="./PageHorizontalNav.tsx"
 ```
 
-### Tertiary nav
+### Tertiary navigation
+
+To add a tertiary navigation component to a page, use the `tertiaryNav` property. The following example passes a string of ‘Navigation’ to `tertiaryNav`, but you can pass fully built-out components, such as navigation.
+
+In this example, `<Masthead>` sets the `display` property equal to `"{{ default: 'stack' }}"` to stack the masthead items and also passes in `"inset={{ default: 'insetXs' }}"` to adjust the inset padding.
 
 ```ts file="./PageTertiaryNav.tsx"
 ```
 
-### With or without fill
+### Uncontrolled nav
+
+When the `isManagedSidebar` property is true, it manages the sidebar open/close state, removing the need to pass both `isNavOpen` into the `<PageSidebar>` and `onNavToggle` into the `<PageHeader>`.
+
+```ts file="./PageUncontrolledNav.tsx"
+```
+
+### Filled page sections
+
+To make a `<PageSection>` fill the available vertical space of the page, set the `isFilled` property equal to true.
+
+By default, the last page section sets `isFilled` equal to true.
 
 ```ts file="./PageWithOrWithoutFill.tsx"
 ```
 
 ### Main section padding
 
-```ts file="./PageMainSectionPadding.tsx"
-```
+To adjust the padding of a `<PageSection>`, you can pass in different values to the `padding` property. These value should be aligned to various breakpoints, including 'default', 'sm', 'md', 'lg', 'xl', '2xl'. 
 
-### Uncontrolled nav
+To remove padding, pass 'noPadding' into `padding`. For example, the second section in this example passes in `padding={{ default: 'noPadding' }}`. Since no specific breakpoints are mentioned, every breakpoint will have 'noPadding'.
 
-```ts file="./PageUncontrolledNav.tsx"
-```
+To add padding on specific breakpoints, specify the value for those breakpoints. For example, the third section in this example passes in `padding={{ default: 'noPadding', md: 'padding', lg: 'padding' }}`. On 'md' and 'lg' breakpoints, the default value will be overwritten, and padding will be added.
```

Comment:
The way this is worded, and the way the modifiers are used currently seems to imply that if you set padding/nopadding at a breakpoint, it applies to _that breakpoint only_. FWIW, as the breakpoints increase (the page viewport width increases), each breakpoint's padding value inherits from the previous breakpoint. In this example, you'll notice that whatever padding is set on `lg` also applies to `xl` and `2xl`. Which means that `md - padding, lg - padding` is redundant - you only need `md - padding` in that example, because `lg` and above will inherit from `md`.
 
The page section in the example that shows this has the text "Section with padding only on medium/large," so I think a good way to show this would be `default - noPadding, md - padding, xl - noPadding`. Though there is a blurb below about removing padding at a breakpoint, so my suggestion here could be out of order with that.